flytekit/clis/sdk_in_container/run.py

```diff
@@ -256,6 +256,16 @@ class RunLevelParams(PyFlyteParams):
             help="Assign newly created execution to a given cluster pool",
         )
     )
+    execution_cluster_label: str = make_click_option_field(
+        click.Option(
+            param_decls=["--execution-cluster-label", "-ecl"],
```

Comment:
```suggestion
            param_decls=["--execution-cluster-label", "--ecl"],
```

Should there be two hyphens? See e.g. `["--envvars", "--env"]`.

Reply:
My intent was to use one similar to how you'd have for `--version` = `-v`? Lmk what you think. `-ecl` [should work too](https://stackoverflow.com/a/75448996).

Reply:
Looking at the existing args ...

```py
param_decls=["--envvars", "--env"],
param_decls=["--raw-output-data-prefix", "--raw-data-prefix"],
param_decls=["-r", "--remote"],
```

... I'd say currently we only use a single hyphen if it is followed by a single character like `-v` or `-r`. If multiple characters follow like `--env`, we use two hyphens.

I don't have strong opinions about this but I'd propose to just stick with `--` here as well.